src/cmd/singularity/cli/pull.go

```diff
@@ -44,19 +44,23 @@ var PullCmd = &cobra.Command{
 	PreRun: sylabsToken,
 	Run: func(cmd *cobra.Command, args []string) {
 		var uri, image string
+
 		image = ""
+
 		if len(args) == 2 {
 			uri = args[1]
 			image = args[0]
 		} else {
 			uri = args[0]
 		}
+
 		BaseURI := strings.Split(uri, "://")
+
 		switch BaseURI[0] {
 		case SyCloudLibrary:
```

Comment:
Technically these are URI protocols.... how about 'ProtocolShub' 'ProtocolLibrary'? I'm not sure we are going to call it SyCloud... lets avoid that kinda branding type name.